e2e-tests/content-strings/pages/insurance/declarations/modern-slavery/index.js

```diff
@@ -0,0 +1,14 @@
+import { SHARED } from '../shared';
+
+export const MODERN_SLAVERY = {
+  ...SHARED,
+  PAGE_TITLE: 'Modern slavery',
+  INTRO: {
+    ANSWER_THE_QUESTIONS: 'Answer the questions about complying with modern slavery legislation and the',
+    GUIDING_PRINCIPLES_LINK: {
+      TEXT: 'UN Guiding Principles on Business and Human Rights',
+      HREF: '#',
```

Comment:
Note: we do not have a link for this, pending BA/design. There is a subtask for this.